lib/adapters/sequelize.js

```diff
@@ -139,7 +139,20 @@ module.exports = Bone => {
     //   throw new Error('unimplemented');
     // }
 
-    static async destroy(options = {}) {
+    static async destroy(options = {}, runOpt = {}) {
+      const { where, individualHooks } = options;
+      if (individualHooks) {
+        const instances = await this.find(where);
+        if (instances.length) {
+          return await Promise.all(instances.map(async (instance) => await instance.destroy(options, runOpt)));
```

Comment:
```suggestion
          return await Promise.all(instances.map((instance) => instance.destroy(options, runOpt)));
```

async/await can be omitted here since there's only one promise to wait